src/physics/em/BetheHeitlerInteractor.hh

```diff
@@ -0,0 +1,121 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file BetheHeitlerInteractor.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Macros.hh"
+#include "base/Types.hh"
+#include "physics/base/Interaction.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "physics/base/Secondary.hh"
+#include "physics/base/SecondaryAllocatorView.hh"
+#include "physics/base/Units.hh"
+#include "physics/material/ElementView.hh"
+#include "BetheHeitlerInteractorPointers.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Bethe-Heitler model for gamma -> e+e- (electron-pair production).
+ *
+ * Give an incident gamma, it adds a two pair-produced secondary electrons to
+ * the secondary stack. No cutoffs are performed on the incident gamma energy.
+ *
+ * \note This performs the same sampling routine as in Geant4's
+ *  G4BetheHeitlerModel, as documented in section 6.5 of the Geant4 Physics
+ *  Reference (release 10.6), applicable to incident gammas with energy
+ *  E_gamma \leq 100 GeV. For E_gamma > 80 GeV, it is suggested to use
```

Comment:
Doxygen will complain about `\leq` since it looks like a documentation command rather than a latex macro.
```suggestion
 *  \f$ E_\gamma \leq 100 GeV \f$.  For \f$ E_\gamma > 80 GeV \f$, it is suggested to use
```